pages/linux/wine.md

```diff
@@ -3,13 +3,13 @@
 > Run Windows programs on Unix.
 > More information: <https://wiki.winehq.org/>.
 
-- Run a program with args:
+- Run ipconfig.exe program:
```

Comment:
I think what others meant is something like this:

```
Run a program with args:
wine {{ipconfig}} {{/all}}
```

Reply:
Yes, that's what I mentioned here: https://github.com/tldr-pages/tldr/pull/3453#issuecomment-565712705. Sorry if that was confusing.

Reply:
Oh yes my mistake!  I corrected my syntax